mappings/net/minecraft/block/Block.mapping

```diff
@@ -389,6 +391,7 @@ CLASS bme net/minecraft/block/Block
 		ARG 1 state
 		ARG 2 view
 		ARG 3 pos
+	METHOD d isSolidFullSquare (Lbvc;Lbgl;Lev;Lfa;)Z
```

Comment:
What is the distinction between small, medium, and full?

Reply:
Small is used for torches, lanterns and pressure plates - it checks against a `VoxelShape` of size 4x4. Medium is used for rails, repeaters/comparators and pressure plates - 12x12. Full is well, a full block - so 1x1.

I'm aware they're not fantastic names, but I couldn't really find a name which covered all the cases they're used.

Reply:
Consider naming them based on their usages then.

Reply:
`isSolidForTorchOrLantern` doesn't really roll off the tongue. IMO it's better to describing what it does, and people can look at the implementation if they need to.

Reply:
How about something like `isHitboxPole`?